tasks.js

```diff
@@ -60,17 +60,49 @@ console.log('Is average correct?: ', avg.toFixed(2) === '68333.33')
 //true
   
   /* Task - Which department is the biggest? */
-const biggestDept: string = '' // write your function here
-  
+// const biggestDept: string = '' // write your function here
+ 
+const marketing = employees.filter(employee => employee.department === "Marketing").length;
+const sales = employees.filter(employee => employee.department === "Sales").length;
+const it = employees.filter(employee => employee.department === "IT").length;
+
+const biggestDept = marketing >= sales && marketing >= it ? "Marketing" : sales >= it ? "Sales" : "IT";
+
 console.log('Is biggest department correct?: ', biggestDept === 'Marketing')
   
   /* Task (fun one) - Create an object that groups employees by department 
   The result should look like this: */
-const departments = {
-    marketing: {
-        deptName: 'Marketing',
-        positions: ['Marketing Specialist', 'Content Creator'], // list all positions once
-        people: [] // list all people that works in this dept
-      }
+
+const departments = {};
+
+employees.forEach(emp => {
```

Comment:
Please don't use abreviations when it's not necessary
```suggestion
employees.forEach(employee => {
```